src/balances/sui.ts

```diff
@@ -50,6 +58,51 @@ export async function pullSuiTokenBalances(
   return provider.getAllBalances({ owner: address });
 }
 
+export async function transferSuiNativeBalance(
+    conn: Connection,
+    privateKey: string,
+    txDetails: SuiTransactionDetails,
+): Promise<any> {
+  const provider = new JsonRpcProvider(conn);
+  const { targetAddress, amount } = txDetails;
+
+  try {
+    const keyPair = buildSuiKeyPairFromPrivateKey(privateKey);
+    const signer = new RawSigner(keyPair, provider);
+    const tx = new TransactionBlock();
+    const [coin] = tx.splitCoins(tx.gas, [tx.pure(amount)]);
+
+    tx.transferObjects([coin], tx.pure(targetAddress));
+    const txBlock = await signer.signAndExecuteTransactionBlock({
+      transactionBlock: tx,
+    });
+
+    return makeTxReceipt(
+      await provider.getTransactionBlock({digest: txBlock.digest, options: { showEffects: true }})
+    );
+  } catch (error) {
+    throw new Error(`Could not transfer native balance to address ${targetAddress}. Error: ${error}`);
```

Comment:
I think I'd either avoid wrapping the error here or I'd add the original stack trace to the wrapped error message like this:

```suggestion
    throw new Error(`Could not transfer native balance to address ${targetAddress}. Error: ${(error as Error)?.stack || error}`);
```

Obfuscating stack traces for no reason makes for a poor developer experience :disappointed: